packages/synapse-react-client/src/components/OrientationBanner/OrientationBanner.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { useTheme } from '@mui/material'
+import FullWidthAlert, {
+  AlertButtonConfig,
+} from '../FullWidthAlert/FullWidthAlert'
+import Illustrations from '../../assets/illustrations'
+
+const OrientationBannerNameStrings = [
+  'Challenges',
+  'CohortBuilder',
+  'DataAccessManagement',
+  'Datasets',
+  'Discussions',
+  'Docker',
+  'Favorites',
+  'Files',
+  'Following',
+  'Projects',
+  'Tables',
+  'Teams',
+  'TrashCan',
+  'Wikis',
+] as const
+export type OrientationBannerName =
+  (typeof OrientationBannerNameStrings)[number]
```

Comment:
Create a list of allowed OrientationBanner names, so that we have a complete list of local storage keys to persist in SWC